.ci/bump-opbeans-ruby.yml

```diff
@@ -46,6 +46,16 @@ sources:
     transformers:
       - trimprefix: "v"
 
+conditions:
+  checkIfVersionDiffers:
+    name: Check if installed version differs
+    kind: shell
+    sourceid: elastic-apm-agent-ruby
+    spec:
+      command: bash .ci/scripts/test-version.sh
```

Comment:
NIT: if `.ci/scripts/test-version.sh` got execution permissions, and it uses the shebang I'd say `bash` is not required

Reply:
https://github.com/elastic/opbeans-ruby/pull/186/commits/1550277ac2460b45977f5ed13795086de0d6768a